src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1937,20 +1937,6 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   </Target>
 
-  <!--
-    ============================================================
-                                    GetTargetFrameworkProperties
-
-    Overrridden by cross-targeting projects to return the set of
-    properties (in the form "key1=value1;...keyN=valueN") needed
-    to build it with the best target for the referring project's
-    target framework.
-
-    The referring project's $(TargetFrameworkMoniker) is passed
-    in as $(ReferringTargetFramework)
-  -->
-  <Target Name="GetTargetFrameworkProperties" />
```

Comment:
Yeah, this makes me nervous. It doesn't help that much (one empty target is no big deal), and might cause breaks (if someone's calling it explicitly).

To be conservative we could minimize the comment to "Obsolete; present only for theoretical backward compatibility".